src/Doctrine/EventSubscriber/TourSubscriber.php

```diff
@@ -90,4 +109,15 @@ private function processTourItem (TaskCollectionItem $taskCollectionItem, bool $
             }
         }
     }
+
+    public function postFlush(PostFlushEventArgs $args) {
+
+        foreach ($this->insertedTours as $insertedTour) {
+            $this->eventBus->handle(new TourCreated($insertedTour));
+        }
+
+        foreach($this->updatedTours as $updatedTour) {
+            $this->eventBus->handle(new TourUpdated($updatedTour));
+        }
```

Comment:
Maybe you could empty `$this->insertedTours` & `$this->updatedTours` at the end of this method to be 100% sure it's clean.